app/app.go

```diff
@@ -1233,6 +1264,22 @@ func (app *App) GetSubspace(moduleName string) paramstypes.Subspace {
 	return subspace
 }
 
+func (app *App) GetStakingKeeper() ibctestingtypes.StakingKeeper {
```

Comment:
Not sure this should be `ibctestingtypes`?

Reply:
This interface was added for ibctesting compatibility, but cw-ica module does not require this interface though since it is not doing any module level txs for ica.